### PR TITLE
jim: Add version 0.9.9.9.7

### PR DESCRIPTION
**Pull request by MatyiFKBT**

### CODE REVIEW

**Comment by MatyiFKBT** [comment]
@Calinou seems like theCI check failed because of another app (toontownrewritten).
I created another PR #224, it should fix that.

File: bucket/jim.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "description": "TF2 Item Manager",
+    "url": "http://www.jengerer.com/item_manager/downloads/item_manager.zip",
+    "homepage": "http://www.jengerer.com/item_manager/",
+    "bin": "item_manager.exe",
+    "shortcuts": [
+        [
+            "item_manager.exe",
+            "Jengerer's Item Manager"
+        ]
+    ]
```
Comment (Calinou) [inline bucket/jim.json:12]: According to its homepage, the Visual C++ 2010 redistributable is required to run the program:

```suggestion
    ],
    "depends": "extras/vcredist2010"
```

File: bucket/jim.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.9.9.9.7",
+    "description": "TF2 Item Manager",
+    "url": "http://www.jengerer.com/item_manager/downloads/item_manager.zip",
+    "homepage": "http://www.jengerer.com/item_manager/",
```
Comment (Calinou) [inline bucket/jim.json:5]: Add a license field:

```suggestion
    "homepage": "http://www.jengerer.com/item_manager/",
    "license": "Proprietary",
```

**Review by Calinou — CHANGES_REQUESTED** [review]

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**